tools/src/commands/EasDispatch.ts

```diff
@@ -124,11 +119,11 @@ async function main(actionId: string | undefined) {
 }
 
 function getAndroidApkUrl(appVersion: string): string {
-  return `https://d1ahtucjixef4r.cloudfront.net/Exponent-${appVersion}.apk`;
+  return `https://github.com/expo/expo-go-releases/releases/download/Exponent-${appVersion}/Exponent-${appVersion}.apk`;
 }
 
 function getIosSimulatorUrl(appVersion: string): string {
-  return `https://dpq5q02fu5f55.cloudfront.net/Exponent-${appVersion}.tar.gz`;
+  return `https://github.com/expo/expo-go-releases/releases/download/Exponent-${appVersion}/Exponent-${appVersion}.tar.gz`;
```

Comment:
while we're doing this, lets also rename this archive from `Exponent-` to `Expo-Go-` before uploading

Reply:
still appears private to me!

<img width="303" height="109" alt="Screenshot 2025-08-13 at 9 12 13 AM" src="https://github.com/user-attachments/assets/3faf603d-711d-4aae-b6c9-7bae926d7e9e" />